addon/store.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const keys = Ember.keys;
+const create = Ember.create;
+export const knownStores = [];
+
+export default DS.Store.extend({
+  init() {
+    this._super(...arguments);
+    knownStores.push(this._debugContainerKey);
```

Comment:
If this is run on the server, wouldn't you expect to init a particular store many times even though you import it once? Probably need to make sure the same key is not pushed multiple times